Kernel_23/test/Kernel_23/test_approximate_dihedral_angle_3.cpp

```diff
@@ -10,10 +10,51 @@ struct query {
   double expected_angle;
 };
 
+void sign_test()
+{
+  K::Point_3 a(0,0,0), b(1,0,0), c(0,1, 0), d(0,0,1);
+
+  assert( CGAL::approximate_dihedral_angle(a, b, c, d) > 0);
+  assert( CGAL::approximate_dihedral_angle(c, a, b, d) > 0);
+  assert( CGAL::approximate_dihedral_angle(a, d, b, c) > 0);
+  assert( CGAL::approximate_dihedral_angle(c, b, d, a) > 0);
+  assert( CGAL::approximate_dihedral_angle(d, b, a, c) > 0);
+  assert( CGAL::approximate_dihedral_angle(d, c, b, a) > 0);
+
+  assert( CGAL::approximate_dihedral_angle(a, b, d, c) < 0);
+  assert( CGAL::approximate_dihedral_angle(c, a, d, b) < 0);
+  assert( CGAL::approximate_dihedral_angle(a, d, c, b) < 0);
+  assert( CGAL::approximate_dihedral_angle(c, b, a, d) < 0);
+  assert( CGAL::approximate_dihedral_angle(d, b, c, a) < 0);
+  assert( CGAL::approximate_dihedral_angle(d, c, a, b) < 0);
+}
+
+auto almost_equal_angle(double a, double b) {
+  return std::min(std::abs(a - b), std::abs(a + 360 - b)) < 0.1;
```

Comment:
```suggestion
  return (std::min)(std::abs(a - b), std::abs(a + 360 - b)) < 0.1;
```

Reply:
Is 0.1  a reasoned choice?

Reply:
(Sorry I forgot about the protection against the `min` macro.)

About `0.1`, that is a small value (for an angle in degrees) and it worked for me. We could try another smaller value, as soon as it passes the tests. I let you modify that value if you want.